lib/commands/commands/riot.ts

```diff
@@ -32,39 +34,149 @@ export const getSummonerId = async (ign:string, server:string) => {
+            } in ${ Math.round(parseInt(lastGameData.data.gameDuration)/60) } minutes.`);
+
+        relevantMinutes.map((minute:number) => {
+            if (lastGameTimeline.data.frames[minute]) {
+                const timeline = lastGameTimeline.data.frames[minute].participantFrames;
+                const player:any = Object.values(timeline).find((player:any) => player.participantId === ourPlayer.gameId);
+
+                gameFrames[`min${minute}`] = {
+                    player: {
+                        cs: player.minionsKilled + player.jungleMinionsKilled,
+                        gold: player.totalGold,
+                        level: player.level
+                    },
+                    enemies: [],
+                    allyTeam: 0,
+                    enemyTeam: 0,
+                };
+
+                enemies.map((enemy:any) => {
+                    const player:any = Object.values(timeline).find((player:any) => player.participantId === enemy.gameId);
+                    gameFrames[`min${minute}`].enemies.push({
+                        id: enemy.gameId,
+                        cs: player.minionsKilled + player.jungleMinionsKilled,
+                        gold: player.totalGold,
+                        level: player.level,
+                    })
+                });
+
+                Object.values(timeline).map((player:any) => {
+                    const currentPlayer = players.find((p:any) => p.gameId === player.participantId);
+                    if (currentPlayer && currentPlayer.teamId !== ourPlayer.teamId && currentPlayer.lane !== ourPlayer.lane)
+                        gameFrames[`min${minute}`].enemyTeam += player.totalGold;
+                    if (currentPlayer && currentPlayer.teamId === ourPlayer.teamId && currentPlayer.lane !== ourPlayer.lane)
+                        gameFrames[`min${minute}`].allyTeam += player.totalGold;
+                });
+
+                const playerChampionName = ourPlayer.champion ? ourPlayer.champion.name : '[unknown ally champion]';
+                const enemyChampionName = enemies[0].champion ? enemies[0].champion.name : '[unknown senemy champion]';
+                const playerGold = gameFrames[`min${minute}`].player.gold;
+                const playerCs = gameFrames[`min${minute}`].player.cs;
+                const enemyCs = gameFrames[`min${minute}`].enemies[0].cs;
+                const enemyGold = gameFrames[`min${minute}`].enemies[0].gold;
+                const allyTeamGoldAdvantage = gameFrames[`min${minute}`].allyTeam - gameFrames[`min${minute}`].enemyTeam;
+                const minuteSummary = gameFrames[`min${minute}`]
+                    ? `${playerChampionName} has **${Math.abs(playerGold - enemyGold)}** gold ${playerGold - enemyGold > 0 ? 'advantage' : 'disadvantage'}.
+                        CS scores are ${playerChampionName}'s **${playerCs}** to ${enemyChampionName}'s **${enemyCs}**.
```

Comment:
Before you were using `\n` and here it's not.

Reply:
I'm not using `\n` here because the backticks preserve whitespaces